src/kernel/src/signal/mod.rs

```diff
@@ -7,42 +7,85 @@ use std::num::NonZeroI32;
+                assert!($num > 0 && $num <= SIG_MAXSIG);
+                NonZeroI32::new_unchecked($num)
+            });
+        )*
+
+        pub fn strsignal_impl(sig: Signal) -> Cow<'static, str> {
+            match sig.0.get() {
+                $( $num => Cow::Borrowed(stringify!($name)), )*
+                _ => format!("{sig}", sig = sig.get()).into(),
+            }
+        }
+    };
+}
+
 // List of PS4 signals. The value must be the same as PS4 kernel.
-pub const SIGHUP: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(1) };
-pub const SIGINT: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(2) };
-pub const SIGQUIT: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(3) };
-pub const SIGILL: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(4) };
-pub const SIGTRAP: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(5) };
-pub const SIGABRT: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(6) };
-pub const SIGEMT: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(7) };
-pub const SIGFPE: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(8) };
-pub const SIGKILL: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(9) };
-pub const SIGBUS: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(10) };
-pub const SIGSEGV: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(11) };
-pub const SIGSYS: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(12) };
-pub const SIGPIPE: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(13) };
-pub const SIGALRM: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(14) };
-pub const SIGTERM: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(15) };
-pub const SIGURG: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(16) };
-pub const SIGSTOP: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(17) };
-pub const SIGTSTP: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(18) };
-pub const SIGCONT: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(19) };
-pub const SIGCHLD: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(20) };
-pub const SIGTTIN: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(21) };
-pub const SIGTTOU: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(22) };
-pub const SIGIO: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(23) };
-pub const SIGXCPU: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(24) };
-pub const SIGXFSZ: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(25) };
-pub const SIGVTALRM: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(26) };
-pub const SIGPROF: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(27) };
-pub const SIGWINCH: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(28) };
-pub const SIGINFO: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(29) };
-pub const SIGUSR1: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(30) };
-pub const SIGUSR2: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(31) };
-pub const SIGTHR: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(32) };
-pub const SIGNONE: NonZeroI32 = unsafe { NonZeroI32::new_unchecked(128) };
-pub const SIG_MAXSIG: i32 = 128;
+#[rustfmt::skip]
```

Comment:
I want one signal per line, I think without this, Ctrl + S tries format it with several per line. I think that's what happened, it's already some time since this PR.

Reply:
Welcome back, by the way! I think there's some things that might need adjusting that were merged, but they should all probably be more or less ok

Reply:
Can you try `signals {` instead of `signals(`? AFAIK rust-analyzer will not format the macro with this syntax.

Reply:
Done